interfaces/ros1/wavemap_ros/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(wavemap_ros)
+
+find_package(catkin_simple REQUIRED)
+catkin_simple(ALL_DEPS_REQUIRED)
```

Comment:
This still required, i.e. you still have a package needing catkin_simple? I've seen you've removed it from a lot of places so far.